doc/source/overview.rst

```diff
@@ -117,6 +117,46 @@ Institutional Partners
 
 The information about current institutional partners can be found on `pandas website page <https://pandas.pydata.org/about.html>`__.
 
+Modules Privacy Has Changed
```

Comment:
Remove the "Has Changed"

Reply:
Sorry, didn't think about that. I'll get right on it.